<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_file_type</class>
 <widget class="QDialog" name="add_file_type">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add File Type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="add_file_text">
       <property name="styleSheet">
        <string notr="true">color:black</string>
       </property>
       <property name="text">
        <string>Add a file type in the format: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_format">
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string>*.xxx</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="le_user_input"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
